documentation/source/user_section/tutorials/shape-analysis/normalizing-morphometrics-compressions/normalization-pipeline.rst

```diff
@@ -0,0 +1,30 @@
+.. _normalization-pipeline:
+
+Normalization pipeline
+######################
+
+Morphometrics measures are output from ``sct_process_segmentation`` and include cross-sectional area (area), anteroposterior diameter (diameter_AP), right-left diameter (diameter_RL), eccentricity and solidity.
+Metrics are normalized using the non-compressed levels above and below the compression site
+using the equation in the figure below.
+
+Additionally, if the ``-normalize-hc 1`` flag is used, metrics are normalized using a database
+built from adult healthy participants. This database uses the PAM50 template as an anatomical
+reference system.
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/sb/4158-add-tutorial-sct-compute-compression/spinalcord-compresssion-norm/normalized_metrics_hc_figure.png
+   :align: center
+
+   Normalized metric ratio using a database of healthy controls.
+
+
+:Legend:
+   - ``mi`` : metric at the compression level.
+   - ``ma`` : metric above the compression levels.
+   - ``mb`` : metric below the compression level.
+
+
+.. note::
+   The MSCC (maximum spinal cord compression) metric was originally introduced in the following study:
+   Miyanji F, Furlan JC, Aarabi B, Arnold PM, Fehlings MG. Acute cervical traumatic
+   spinal cord injury: MR imaging findings correlated with neurologic outcome--prospective
+   study with 100 consecutive patients. Radiology 2007;243(3):820-827.
```

Comment:
```suggestion
```

I suggest removing because (i) the citation now appears earlier https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4162/files#r1317729773, and (ii) in the rest of the tutorial, citations appear as hyperlinks directly in the text, rather than as notes